planship/setup.py

```diff
@@ -18,18 +18,24 @@
     "planship_openapi_gen >= 1",
 ]
 
+project_urls = {
+    "Repository": "https://github.com/planship/planship-python",
+    "Documentation": "https://docs.planship.io",
```

Comment:
Do you think people might assume that "documentation" means docs for this client and not Planship docs in general?

Reply:
I think this is fine, especially since I added a link to the class reference here. The project home link also points to Planship itself (marketing page).

Reply:
I think "Planship Docs" would be clearer but up to you.